conda/cli/main_export.py

```diff
@@ -152,10 +176,16 @@ def execute(args: Namespace, parser: ArgumentParser) -> int:
     prefix = context.target_prefix
 
     # Create models.Environment directly
+    # TODO: Figure out how to handle source and target platforms.  Do we
+    #       we need to specify the source platform and then export to
+    #       the target platform?  If so, is this done in the
+    #       environment_exporter?
+    # target_platform = target_platforms[0] if target_platforms else context.subdir
+
     env = Environment.from_prefix(
         prefix=prefix,
         name=env_name(prefix),
-        platform=context.subdir,
+        platform=target_platforms[0],
```

Comment:
Hey, @ryanskeith ! 
Why are we doing `target_platforms[0]` here?

Reply:
This PR only adds the `--platform` CLI argument and `export_platforms` context variable, the export command doesn't support multiple platforms yet